app/services/assurance_reports/query.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module AssuranceReports
+  class Query
+    def initialize(statement)
+      self.statement = statement
+    end
+
+    def declarations
+      Declaration.find_by_sql(sql)
+    end
+
+  private
+
+    attr_accessor :statement
+
+    def sql
+      <<~EOSQL
+        SELECT
+          d.id                                    AS id,
+          d.ecf_id                                AS ecf_id,
```

Comment:
Yes, your right - I'd added it because `id` no longer matches the 'public facing id' but your right its already there in `declaration_id` :+1: 

I'll update the PR